src/surabhi/BankDetails.java

```diff
@@ -0,0 +1,74 @@
+package surabhi;
+
+/*Create a Banking System which has the following functionality.
+
+Create two object to perform below scenario.
+1) Method to debit amount
+2) Method to credit amount
+3) Method to printBalance [it will print current balance]
+4) individualTransactionSummary() method should show how many times individually debit, credit & printBalance method called .
+Hint : Create 3 non static instance variable.
+5) allTransactionSummary() method should show by both users total how many times debit, credit & printBalance method get called.
+Hint : Create 3 static variables will be required.
+*/
+
+public class BankDetails {
+	String Customername;
+	int currentBal=10000;
+	int debitCount=0, creditCount=0, counter=0;
+	static int debitAmtCount=0, creditAmtCount=0, methodCounter=0;
+	
+	void debitAmt(int amt, String name) {
+		Customername=name;
```

Comment:
why this require here?

Reply:
to Know user name as we created two objects, I hv created method with debit amount and user name in parameter